python/rec_multi_reads.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+import __future__
+import nibabel as nib
+import numpy as np
+import time
+
+# image shape: 3850 3025 3500
+# file name format: bigbrain_3080_1815_1400.nii.gz
+
+
+# info of header. Code should be modified before merging to imageutils.py
```

Comment:
rec_multi_reads.py should be removed as it's in the imageutls.py now and I forgot to do that, sry.